### PR TITLE
Add FAQ about chat history and albums

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Those are probably two of the most frequently asked questions, so it is well deserved to have them in the list of FAQ.

### CODE REVIEW

File: site/docs/resources/faq.md [hunk 1]
```diff
@@ -26,6 +27,42 @@ No, it's not.
    As always, TypeScript will auto-complete the parameter names for you.
 3. Perhaps double-check the method signature for [actions](../guide/context#available-actions) on `ctx` [here](https://deno.land/x/grammy/mod.ts?s=Context#Methods), or for API methods (`ctx.api`, `bot.api`) [here](https://deno.land/x/grammy/mod.ts?s=Api#Methods).
 
+## How Can I Access the Chat History?
+
+You can't.
+
+Telegram does not store the messages for your bot.
```
Comment (roj1512) [inline site/docs/resources/faq.md:34]: That as it is phrased right now, is right on the high level, but also a lie if you take it seriously. Would you like to go with it?

Comment (roj1512) [inline site/docs/resources/faq.md:34]: This as it is phrased right now, is right on the high level, but also a lie if you take it seriously. Would you like to go with it?

**Comment by roj1512** [comment]
image

File: site/docs/resources/faq.md (hunk 1 above)
Comment (KnorpelSenf) [reply site/docs/resources/faq.md:34]: I think that the people who read this basic question do actually need to hear a simplified view of the things. Otherwise they start hacking around with `forwardMessage` to private channels etc, and those are big hacks that don't scale well and potentially get people banned.

How about this?
```suggestion

Telegram does not store the messages for your bot (although messages are of course stored for the user).
```

**Comment by KnorpelSenf** [comment]
It seems like SSH signing is not really working, the key is clearly there. I'll try to fix this.

File: site/docs/resources/faq.md (hunk 1 above)
Comment (roj1512) [reply site/docs/resources/faq.md:34]:
```suggestion

Telegram does not let bots see chat histories.
```

How about this one?

**Comment by roj1512** [comment]
I'm using SSH signing, and it works fine for me.

File: site/docs/resources/faq.md [hunk 2]
```diff
@@ -26,6 +27,42 @@ No, it's not.
...
+There is no way to obtain the data, because it is not there.
+
```
Comment (roj1512) [inline site/docs/resources/faq.md:36]: Need to remove this too if we accepted my suggestion, and maybe rephrase the sentences after that. Only if we accepted my suggestion tho.

File: site/docs/resources/faq.md (hunk 1 above)
Comment (KnorpelSenf) [reply site/docs/resources/faq.md:34]: That is more confusing than not saying this at all. It sounds like they just have some obscure reason why they don't reveal the data. But that is not the case, is it?

**Comment by KnorpelSenf** [comment]
> I'm using SSH signing, and it works fine for me.

I can verify my commits locally with git, and I have uploaded the public SSH key to GH … what am I missing?

**Comment by KnorpelSenf** [comment]
I fixed it! Thanks go to @wojpawlik

File: site/docs/resources/faq.md [hunk 3]
```diff
@@ -26,6 +27,42 @@ No, it's not.
    As always, TypeScript will auto-complete the parameter names for you.
 3. Perhaps double-check the method signature for [actions](../guide/context#available-actions) on `ctx` [here](https://deno.land/x/grammy/mod.ts?s=Context#Methods), or for API methods (`ctx.api`, `bot.api`) [here](https://deno.land/x/grammy/mod.ts?s=Api#Methods).
 
+## How Can I Access the Chat History?
+
+You can't.
+
+Telegram does not store the messages for your bot.
+There is no way to obtain the data, because it is not there.
```
Comment (roj1512) [inline site/docs/resources/faq.md:35]: What about removing this this?

```suggestion
```

**Review by roj1512 — APPROVED** [review]

File: site/docs/resources/faq.md (hunk 3 above)
Comment (KnorpelSenf) [reply site/docs/resources/faq.md:35]: Works!

**Comment by roj1512** [comment]
@KnorpelSenf Feel free to add the "ready for translation" label after ignoring or applying the last suggestion from me.

**Comment by KnorpelSenf** [comment]
Done

File: site/docs/resources/faq.md [hunk 4]
```diff
@@ -26,6 +27,41 @@ No, it's not.
...
+Telegram does not store the messages for your bot.
+
+Instead, you need to wait for new messages/channel posts to arrive, and store the messages in your database.
+You can then load the chat history from your database.
+
+This is what [conversations](../plugins/conversations) do internally for the relevant part of the message history.
+
+## How Can I Handle Albums?
+
+You can't ... at least not in the way you think.
```
Comment (KnorpelSenf) [inline site/docs/resources/faq.md:43]: Does the build step turn this into …? Else we should fix it

Comment (KnorpelSenf) [reply site/docs/resources/faq.md:43]: It does, nvm

Comment (roj1512) [reply site/docs/resources/faq.md:43]: I don’t think so. The typographer plugin does that when rendering.

File: site/docs/uk/resources/faq.md [hunk 5]
```diff
@@ -26,6 +27,42 @@ prev:
...
+## Як я можу обробляти альбоми?
+
+Ви не можете...
+Принаймні, не так, як ви думаєте.
+
+Альбом реально існує лише в інтерфейсі клієнта Telegram.
+Для бота робота з медіа-групою --- це те саме, що робота з серією окремих повідомлень.
+Найпрактичніша порада --- ігнорувати існування медіа-груп і просто створювати бота для роботи з окремими повідомленнями.
+Тоді альбоми працюватимуть автоматично.
+Наприклад, ви можете попросити користувача [натиснути кнопку](../plugins/keyboard.md#inline-клавіатури) або надіслати `/done`, коли всі файли буде завантажено до чату вашого бота.
```
Comment (niusia-ua) [inline site/docs/uk/resources/faq.md:50]:
```suggestion
Наприклад, ви можете попросити користувача [натиснути кнопку](../plugins/keyboard.md#вбудовані-клавіатури) або надіслати `/done`, коли всі файли буде завантажено до чату вашого бота.
```

**Review by LWJerri — APPROVED** [review]

File: site/docs/uk/resources/faq.md (hunk 5 above)
Comment (LWJerri) [reply site/docs/uk/resources/faq.md:50]: Accept it now or later when it's ready.

**Review by deptyped — APPROVED** [review]

File: site/docs/resources/faq.md (hunk 4 above)
Comment (KnorpelSenf) [reply site/docs/resources/faq.md:43]: That is part of the build step, it's what I was referring to :)

**Comment by quadratz** [comment]
@dzikrl btw, I checked out https://pasti.kemdikbud.go.id/, and it says the term 'frequently asked questions (FAQ)' is translated as 'soal sering ditanya (SSD)'. But that term isn't familiar to me. What do you think?

**Review by quadratz — CHANGES_REQUESTED** [review]
@dzikrl btw, I checked out https://pasti.kemdikbud.go.id/, and it says the term `frequently asked questions (FAQ)` is translated as `soal sering ditanya (SSD)`. But that term isn't familiar to me. What do you think?

**Review by habemuscode — APPROVED** [review]

**Comment by roj1512** [comment]
@quadratz Should we add ID label?

**Comment by quadratz** [comment]
Not yet. Feel free to add them and resolve the pending review if there is no answer from @dzikrl in the next 3 days.

File: site/docs/id/resources/faq.md [hunk 6]
```diff
@@ -26,6 +27,41 @@ Tentu saja tidak.
...
+
+Cara yang sama juga digunakan oleh [plugin percakapan (conversations)](../plugins/conversations) untuk menyimpan riwayat pesan terkait.
+
+## Bagaimana Cara Menangani Album?
+
+Tidak bisa ... setidaknya tidak seperti yang kamu bayangkan.
+
+Album --- _yang mana merupakan bagian dari kelompok media (media group)_ --- sebenarnya berasal dari tampilan UI aplikasi Telegram.
+Di sisi lain, bot menangani kelompok media sebagai runtutan pesan, alih-alih sebagai satu-kesatuan.
+Cara yang paling praktis untuk mengatasinya adalah anggap kelompok media tidak pernah ada, sehingga kamu perlu menulis bot dengan pola pikir tiap pesan sebagai individu, bukan kelompok.
```
[the remark this answers is not in the document]
Comment (quadratz) [reply site/docs/id/resources/faq.md:47]: LGTM. That sounds more concise.

[the remark this answers is not in the document]
Comment (quadratz) [reply site/docs/id/resources/faq.md]: Literal translation of "mindset". Perhaps you could find a better phrasing?

File: site/docs/id/resources/faq.md [hunk 7]
```diff
@@ -26,6 +27,41 @@ Tentu saja tidak.
...
+
+Cara yang sama juga digunakan oleh [plugin percakapan (conversations)](../plugins/conversations) untuk menyimpan riwayat pesan terkait.
+
+## Bagaimana Cara Menangani Album?
+
+Tidak bisa ... setidaknya tidak seperti yang kamu bayangkan.
+
+Album --- _yang mana merupakan bagian dari kelompok media (media group)_ --- sebenarnya berasal dari tampilan UI aplikasi Telegram.
+Di sisi lain, bot menangani kelompok media sebagai runtutan pesan, alih-alih sebagai satu-kesatuan.
+Cara yang paling praktis untuk mengatasinya adalah dengan menganggap kelompok media tersebut tidak pernah ada, sehingga kamu perlu menulis bot dengan pola pikir tiap pesan sebagai individu, bukan kelompok.
```
[the remark this answers is not in the document]
Comment (quadratz) [reply site/docs/id/resources/faq.md:47]: That sounds confusing, what about this one
```suggestion
Cara yang paling praktis untuk mengatasinya adalah mengabaikan keberadaan kelompok media tersebut.
Artinya, kita menulis bot dengan menganggap setiap pesan sebagai individu, bukan kelompok.
```

**Comment by quadratz** [comment]
> Edit : I think "Pertanyaan Umum" != "Pertanyaan yang Sering Diajukan".

Yes. They are different things.

Since "SSD" is not commonly used, we will leave them as "FAQ" for now. If we want to change them for real, we should make separate PQ since it's now out of topic.

File: site/docs/zh/resources/faq.md [hunk 8]
```diff
@@ -26,6 +27,41 @@ prev:
...
+
+## 我该如何处理相册？
+
+你不能，至少不能像你想的那样。
+
+相册只存在于 Telegram 客户端的 UI 中。
+对于一个 bot 来说，处理媒体组与处理一系列单条消息是一回事。
+最实用的建议是忽略媒体组的存在，只需在编写你的 bot 时按照单条消息处理即可。
+这样，相册就会自动生效。
+例如，你可以让用户在所有文件都上传到你的 bot 之后 [点击一个按钮](../plugins/keyboard.md#inline-keyboards) 或者发送 `/done`。
```
Comment (quadratz) [inline site/docs/zh/resources/faq.md:49]: Invalid path

File: site/docs/uk/resources/faq.md [hunk 9]
```diff
@@ -26,6 +27,42 @@ prev:
...
+## Як я можу обробляти альбоми?
+
+Ви не можете...
+Принаймні, не так, як ви думаєте.
+
+Альбом реально існує лише в інтерфейсі клієнта Telegram.
+Для бота робота з медіа-групою --- це те саме, що робота з серією окремих повідомлень.
+Найпрактичніша порада --- ігнорувати існування медіа-груп і просто створювати бота для роботи з окремими повідомленнями.
+Тоді альбоми працюватимуть автоматично.
+Наприклад, ви можете попросити користувача [натиснути кнопку](../plugins/keyboard.md#вбудовані-клавіатури) або надіслати `/done`, коли всі файли буде завантажено до чату вашого бота.
```
Comment (quadratz) [inline site/docs/uk/resources/faq.md:50]: No file extension, please.

File: site/docs/es/resources/faq.md [hunk 10]
```diff
@@ -27,6 +28,41 @@ No, no es así.
...
+
+## ¿Cómo puedo manejar los álbumes?
+
+No puedes... al menos no de la forma que piensas.
+
+Un álbum sólo existe realmente en la interfaz de usuario de un cliente de Telegram.
+Para un bot, manejar un grupo de medios es lo mismo que manejar una serie de mensajes individuales.
+El consejo más práctico es ignorar que existen los grupos de medios, y simplemente escribir tu bot con mensajes individuales en mente.
+Entonces, los álbumes funcionarán automáticamente.
+Por ejemplo, puedes pedir al usuario que [pulse un botón](../plugins/keyboard.md#teclados-en-linea) o envíe `/done` cuando todos los archivos estén subidos al chat de tu bot.
```
Comment (quadratz) [inline site/docs/es/resources/faq.md:50]: No file extension, please.

**Review by quadratz — CHANGES_REQUESTED** [review]

File: site/docs/zh/resources/faq.md (hunk 8 above)
Comment (KnorpelSenf) [reply site/docs/zh/resources/faq.md:49]:
```suggestion
例如，你可以让用户在所有文件都上传到你的 bot 之后 [点击一个按钮](../plugins/keyboard#inline-keyboards) 或者发送 `/done`。
```

File: site/docs/uk/resources/faq.md (hunk 9 above)
Comment (KnorpelSenf) [reply site/docs/uk/resources/faq.md:50]:
```suggestion
Наприклад, ви можете попросити користувача [натиснути кнопку](../plugins/keyboard#вбудовані-клавіатури) або надіслати `/done`, коли всі файли буде завантажено до чату вашого бота.
```

File: site/docs/es/resources/faq.md (hunk 10 above)
Comment (KnorpelSenf) [reply site/docs/es/resources/faq.md:50]:
```suggestion
Por ejemplo, puedes pedir al usuario que [pulse un botón](../plugins/keyboard#teclados-en-linea) o envíe `/done` cuando todos los archivos estén subidos al chat de tu bot.
```

**Review by roj1512 — APPROVED** [review]

**Review by quadratz — APPROVED** [review]
